RDF-Pipeline/t/helpers/compare-results.perl

```diff
@@ -74,30 +74,45 @@ sub CompareFiles
 @_ == 2 || die;
 my ($expectedFiles, $resultFiles) = @_;
 # my $cmd = "diff -b -w $quiet -x lm -x ont -x '.*' '$expectedFiles' '$resultFiles'";
-my $cmd = "diff -b -w $quiet '$expectedFiles' '$resultFiles'";
+# my $cmd = "diff -b -w $quiet '$expectedFiles' '$resultFiles'";
 # warn "cmd: $cmd\n";
 ###### TODO: Change this to use the saved Content-type associated with
 ###### the files, as described in issue-53:
 ###### http://code.google.com/p/rdf-pipeline/issues/detail?id=53
 ###### Or maybe implement smarter RDF sniffing?
-# Don't try to compare as Turtle if the files are identical anyway:
-if (-f $expectedFiles && -f $expectedFiles) {
-	`cmp '$expectedFiles' '$resultFiles'`;
-	return 0 if !$?;
-	}
-my $isTurtle = (&IsTurtle($expectedFiles) && &IsTurtle($resultFiles));
-if ($isTurtle) {
-	$cmd = "rdfdiff";
-	$cmd .= " -b" if $quiet;
-	$cmd .= " -f turtle -t turtle '$expectedFiles' '$resultFiles'";
-	}
-print "$cmd\n" if $debug;
-my $output = `$cmd`;
-if ($?) {
-	print "$cmd\n";
-	print $output;
-	return 1;
-	}
+if (-f $expectedFiles && -f $resultFiles) {
+    # First try a standard diff of the two files:
+    `cmp '$expectedFiles' '$resultFiles'`;
+    return 0 if !$?;  # return success if standard diff worked
+
+    # If the diff failed, then sniff to see if this might be a pair of turtle files
+    my $tmpFile = &CommentHttpHeaders($resultFiles); # comment out any headers
```

Comment:
For the record, after discussion I think there is no need to change this at this point.  We could revisit and perhaps use curl -D at some point in the future, but the check for headers is safer than I realized, so I think this is fine for now.